Harvest.Core/Services/EmailService.cs

```diff
@@ -608,6 +610,24 @@ public async Task<bool> CloseoutConfirmation(Project project)
             }
 
             return true;
+        }
+
+        public async Task<int> ResendCloseoutNotifications()

```

Comment:
probably should just be `SendCloseoutNotifications()` because the service itself doesn't know why we are sending them.